src/routes/adminRoutes.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const router = express.Router();
+const adminController = require('../controllers/adminController');
+
+// Define the admin routes
+router.post('/create-user', adminController.verifyAdmin, adminController.createUser);
```

Comment:
I think these should be in `userRoutes` as:
```
POST /user                     // Create a new user object, re-used for signup
DELETE /user/:uid          // Delete a user object, probably from a button on the dashboard
```
See comment on userRoutes.js for more details. There's no need for `/admin/xxx` here, we should just use the verifyAdmin middleware to restrict them.

Reply:
The team is on board with deleting the create students functionality, so I deleted the function to create a new student, and the function to delete students, so these routes should not be here anymore.